source/Hyperspace-Model-Tests/ZnUrlTest.class.st

```diff
@@ -7,6 +7,16 @@ Class {
 	#category : #'Hyperspace-Model-Tests-Zinc'
 }
 
+{ #category : #tests }
+ZnUrlTest >> testAsHostedAt [
+
+	self
+		assert:
+			( 'https://api.example.com:1122/pets?name=fidus' asUrl
+				asHostedAt: 'http://alternative.server.org:4874/reference?no=true' asUrl )
+		equals: 'https://alternative.server.org:4874/pets?name=fidus' asUrl
+]
```

Comment:
@mtabacman can you include some tests involving URLs without ports against URLs with ports?